### ISSUE TITLE
[Feature request] Add ability to add unjumped wormholes

### ISSUE THREAD

**Issue by Darkworlds** [goryn-clade/pathfinder#159]
Just wanted to ask if it would be possible to implement a way to add unjumped wormholes to the graphical interface of pathfinder? To be able to designate the type of hole, add whether connection EOL etc?

If this could be added that would be amazing!

Thanks,

**Comment by TyrHeimdalEVE** [comment]
I totally agree, it would be great if they initialized as unfilled wormholes when populated in pasted sigs, and further updated as connection is selected with enough information to identify type. Tripwire has a similar implementation.

**Comment by ehawman** [comment]
This is the single largest pain point when transitioning from GF to PF. Would _really_ appreciate the ability to see and annotate unpushed holes.
1. As already mentioned, beginning the EOL clock in a timely manner or warning of a crit hole are two hugely valuable pieces.
2. Let's say I'm looking for a C3. In GF I look at the map and can see all unpushed candidates (be they marked C3s, Unknown K162s, or unmarked holes). With PF I must click each system in the chain to see if it has an unpushed WH sig.

I know there are underlying structural differences with how PF and GF handle systems/signatures/connections. GF asks the user which sig they jumped through to link up the placeholder hole with the user's new current system, and a similar workflow would likely be necessary to implement this change.

I know that I categorically do not possess the programming knowledge necessary to accomplish this task, but the benefits of the feature are myriad and I would really like to see it implemented. Is there any way I can contribute? Some kind of feature bounty perhaps?

**Comment by viniciusdel** [comment]
This would still be an amazing feature and the reason our corp doesn't switch to PF.
